test/mutable/index.js

```diff
@@ -73,6 +79,32 @@ describe('Mutable Data', () => {
             })
     );
 
+    it('throws error if custom public is created with name not equal to 32 bytes', async () => {
+      const name = 'non XOR name';
+      const test = () => app.mutableData.newPublic(name, TYPE_TAG);
+      should(test).throw(errConst.XOR_NAME.msg(32));
+    });
+
+    it('throws error if custom private is created with name not equal to 32 bytes', async () => {
```

Comment:
This seems duplicate of test: https://github.com/maidsafe/safe_app_nodejs/pull/208/files#diff-de3a919540b70d521ff29a18e30c207aR45